lib/AST/ASTDemangler.cpp

```diff
@@ -22,14 +22,14 @@
 #include "swift/AST/ASTDemangler.h"
 
 #include "swift/AST/ASTContext.h"
-#include "swift/AST/ClangModuleLoader.h"
 #include "swift/AST/Decl.h"
 #include "swift/AST/GenericSignature.h"
 #include "swift/AST/GenericSignatureBuilder.h"
 #include "swift/AST/Module.h"
 #include "swift/AST/NameLookup.h"
 #include "swift/AST/Type.h"
 #include "swift/AST/Types.h"
+#include "swift/ClangImporter/ClangImporter.h"
```

Comment:
This would cause a cyclic link dependency; please undo this. (That's mostly what ClangModuleLoader is for.)

Reply:
I agree that ClangModuleLoader would be more appropriate and I'm happy to fix this!
(Note that this is just reverting to the state of before my changes, cf. https://github.com/apple/swift/blob/fee5bc477d6d6e46b296717f898f512ca4c92683/lib/AST/ASTDemangler.cpp from February)

Reply:
https://github.com/apple/swift/pull/26674